First draft of getting stated section (#1)
oduameh/hydra: docs/docs/getting-started.md

# Getting started

> To get started quickly, let’s walk you through our standard demo setup, which consists of:

Reviewer: ```suggestion
To get started quickly, we'll walk you through the standard demo setup, which includes:
```

* A cluster of three `hydra-node` directly connected to each other, each having access to one of three Hydra credentials: `Alice`, `Bob`, or `Carol`;
* A single `cardano-node` producing blocks used as a (very fast) local `devnet`;

> * The `hydra-tui` example clients to interact with the individual `hydra-node`.

Reviewer: ```suggestion
* A cluster of three `hydra-nodes`, each directly connected to the others and configured with one of three Hydra credentials: `Alice`, `Bob`, or `Carol`
* A single `cardano-node` producing blocks used as a (very fast) local `devnet`
* The `hydra-tui` example for clients to interact with the individual `hydra-node`.
```

> This tutorial uses [Docker](https://www.docker.com/get-started) to install the nodes, so ensure Docker is installed. If you want to explore alternative ways of running the tools, see the [variant tutorial](https://hydra.family/head-protocol/unstable/docs/getting-started-without-docker) or the [testnet tutorial](https://hydra.family/head-protocol/unstable/docs/tutorial/), which uses pre-built binaries. The documentation pages on [installation](https://hydra.family/head-protocol/unstable/docs/installation) and [configuration](https://hydra.family/head-protocol/unstable/docs/configuration) provide more details.

Reviewer: ```suggestion
This tutorial uses [Docker](https://www.docker.com/get-started) to install the nodes, so ensure Docker is installed. If you want to explore alternative ways of running the tools, see the [variant tutorial](./getting-started-without-docker.md) or the [testnet tutorial](./tutorial/index.md), which uses pre-built binaries. The documentation pages on [installation](./installation) and [configuration](./configuration) provide more details.
```

> Additionally, the `hydra-tui` uses the HTTP/WebSocket API provided by the `hydra-node` behind the scenes. The [testnet tutorial](https://hydra.family/head-protocol/unstable/docs/tutorial/) will show how to use this API using low-level commands, or you can see the [API reference](https://hydra.family/head-protocol/unstable/api-reference) for more details.

Reviewer: ```suggestion
Additionally, the `hydra-tui` uses the HTTP/WebSocket API provided by the `hydra-node` behind the scenes. The [testnet tutorial](./tutorial/index.md) will show how to use this API using low-level commands, or you can see the [API reference](https://hydra.family/head-protocol/unstable/api-reference) for more details.
```

> 

Reviewer: Replace the info shortcut copy with:

'For convenience, we provide a script `./run-docker.sh`, which combines all the following steps. It also performs a few sanity checks to prevent potential issues.

## Set up the devnet

Next, prepare the devnet configuration to bootstrap a local Cardano blockchain. 

> **Note** we use a simple variant of Cardano that requires no stake pools whatsoever.

Reviewer: ```suggestion
**Note** that we use a simplified variant of Cardano that does not require any stake pools.
```

```shell
./prepare-devnet.sh
```

Bring the Cardano node up with:

```shell
docker compose up -d cardano-node
```

Verify that the devnet is up-and-running by checking the logs with:
```shell
docker compose logs cardano-node -f
```

You should see traces containing `TraceAdoptedBlock`, which means that the devnet is producing blocks .. nice!

:::info
Do not wait too long between these two commands. If you get `TraceNoLedgerView` errors from the Cardano node, the genesis start time is too far in the past, and you need to update them by running `prepare-devnet.sh` again.
:::

> Next, we need to give Alice, Bob, and Carol some UTXOs to commit and some ADA to pay fees. We include a script, `seed-devnet.sh`, that uses the `cardano-cli` in the already running `cardano-node` container to do that:

Reviewer: ```suggestion
Next, we need to give Alice, Bob, and Carol some UTXOs for committing and ada for paying fees. To do this, use the `seed-devnet.sh` script, which uses the `cardano-cli` within the already running `cardano-node` container:
```

```shell
./seed-devnet.sh
```

## Start Hydra nodes

Finally, now that the on-chain preparations are done, we can bring the Hydra network (all three nodes for Alice, Bob, and Carol) up by running:

```shell
docker compose up -d hydra-node-{1,2,3}
```

> Using compose, you can start the demo Terminal-based User Interface (a.k.a. `hydra-tui`) to interact with Hydra nodes. There are 3 preconfigured TUI services in the Compose definition: `hydra-tui-1`, `hydra-tui-2`, and `hydra-tui-3`. To connect to the first Hydra node in a terminal, run the following command:

Reviewer: ```suggestion
Using `compose`, you can start the demo Terminal-based User Interface (`hydra-tui`) to interact with Hydra nodes. There are three preconfigured TUI services in the `compose` definition: `hydra-tui-1`, `hydra-tui-2`, and `hydra-tui-3`. To connect to the first Hydra node in a terminal, run the following command:
```
